packages/theme-chalk/src/table.scss

```
@@ -393,6 +386,15 @@
     }
   }
 
+  @include e((footer-wrapper)) {
+    overflow: hidden;
+
+    & tfoot td.#{$namespace}-table__cell {
```

```suggestion
    tfoot td.#{$namespace}-table__cell {
```
`&` doesn't seem to be needed.